docs/dev_plan.md

```diff
@@ -0,0 +1,339 @@
+experiences gained with OpenFAST, particularly its nonlinear beam-dynamics module,
+[BeamDyn](https://github.com/OpenFAST/openfast/tree/main/modules/beamdyn).
+
+For time integration of the index-3 DAEs we will leverage the generalized-alpha
+algorithm now established in BeamDyn which allows for user-controlled numerical
+damping. See Arnold and Brüls (2007) for details.
+
+The primary model for the turbine blades and the tower, which are slender and flexible
+structures, will be nonlinear finite elements based on the geometrically exact beam
+theory (GEBT), which includes bend-twist coupling. Spatial discretization
+will be based on the Legendre spectral finite elements (LSFEs; like
+those in BeamDyn), but with the explicit ability to use the subset of 2-node
+finite elements (a special case of LSFEs).
+
+The choice of rotation representation is being determined. BeamDyn relies on the
+Wiener-Milenkovic vectorial rotation parameterization (three rotational degrees
+of freedom), which contains singularities. The team is currently investigating the
+use of [quaternions](https://en.wikipedia.org/wiki/Quaternion), which are
+singularity free but require four parameters to represent rotation, i.e., they do
+not form a minimum set.
+
+## Verification and validation cases
+
+The list of verification and validation cases is a work in progress. By way of
+semantics, verification cases are those for which an analytical solution exists and
+formal accuracy studies can be examined.  Validation cases are those for which
+we have solutions that are deemed to be better representations of reality.  For
+example, validation results might be from experiments or from higher-fidelity
+numerical simulations (e.g., shell or solid finite element models).
+
+The list of verification and validation cases is a work in progress. By way of 
+semantics, verification cases are those for which an analytical solution exists and 
+formal accuracy studies can be examined.  Validation cases are those for which
+we have solutions that are deemed to be better representations of reality.  For 
+example, validation results might be from experiments or from higher-fidelity 
+numerical simulations (e.g., shell or solid finite element models). 
+
+**Verification cases**
+- Rigid-body dynamics: three-dimensional pendulum
+- Cantilever-beam nonlinear static roll up
+
+**Validation cases**
+- Princeton beam experiment
+- Twisted and curved beams where the benchmarks are highly resolved solid-element Ansys models
+- IEA 15-megawatt turbine where benchmark is a highly resolved shell-element Ansys model
+
+## Target baseline turbines
+
+- [NREL 5-megawatt reference turbine](https://www.nrel.gov/docs/fy09osti/38060.pdf)
+- [IEA 15-megawatt reference turbine](https://github.com/IEAWindTask37/IEA-15-240-RWT)
+
+## High-level development timeline
+
+**CY23 Q2**: The OpenTurbine team will implement a rigid-body dynamics solver following the
```

Comment:
Out of curiosity, why are these written in terms of CY (calendar year)? I've never seen planning or funding aligned with CY, always FY.

Reply:
I see this was added in CY, then changed to FY, and again back to CY. @michaelasprague can you confirm the correct quarters for these milestones? If you think "FY" is unclear to anyone, It may be good to note that the FY is October through September.